app/services/location_lookup_service.rb

```diff
@@ -34,11 +34,6 @@ def error
     response[:error]
   end
 
-  def lower_tier_area_name
```

Comment:
I'm so sorry to do this to you, but it looks like we're going to need this method in this ticket for devolved nations: https://trello.com/c/Tz6It6TX/991-improve-content-for-devolved-nations, so can we not delete this method after all? So sorry!

Reply:
Have removed the commit that deleted it :)